cmd/snap/cmd_interfaces_test.go

```diff
@@ -57,7 +57,7 @@ func (s *SnapSuite) TestConnectionsZeroSlotsOnePlug(c *C) {
 		"Slot  Plug\n" +
 		"-     keyboard-lights:capslock-led\n"
 	c.Assert(s.Stdout(), Equals, expectedStdout)
-	c.Assert(s.Stderr(), Equals, "")
+	c.Assert(s.Stderr(), Equals, "\nThe 'snap interfaces' command is deprecated, try the new 'snap connections'.\n")
```

Comment:
```suggestion
	c.Assert(s.Stderr(), Equals, "\nThe 'snap interfaces' command is deprecated, try the new 'snap connections' command.\n")
```

Reply:
I'm always over minimal in these situations, as I think the user simply needs to quickly scan the facts - I'd even remove *new* (because when is new?), and *try* because we need to be confident. We can always explain more on the forum.
I'd put:
`The 'snap interfaces' command is deprecated. Use 'snap connections' instead.`